SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -561,43 +561,57 @@ void USpatialSender::SendAuthorityIntentUpdate(const AActor& Actor, VirtualWorke
 	if (NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID))
 	{
 		// Also notify the enforcer directly on the worker that sends the component update, as the update will short circuit
-		NetDriver->LoadBalanceEnforcer->QueueAclAssignmentRequest(EntityId);
+		NetDriver->LoadBalanceEnforcer->MaybeQueueAclAssignmentRequest(EntityId);
 	}
 }
 
-void USpatialSender::SetAclWriteAuthority(const Worker_EntityId EntityId, const PhysicalWorkerName& DestinationWorkerId)
+void USpatialSender::SetAclWriteAuthority(
+	const Worker_EntityId EntityId,
+	const PhysicalWorkerName& DestinationWorkerId,
+	const WorkerRequirementSet& ReadAcl,
+	const WorkerRequirementSet& ClientRequirementSet,
+	const TArray<Worker_ComponentId>& ComponentIds)
 {
 	check(NetDriver);
-	check(NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID));
-
-	EntityAcl* EntityACL = NetDriver->StaticComponentView->GetComponentData<EntityAcl>(EntityId);
-	check(EntityACL);
 
 	const FString& WriteWorkerId = FString::Printf(TEXT("workerId:%s"), *DestinationWorkerId);
 
-	WorkerAttributeSet OwningWorkerAttribute = { WriteWorkerId };
+	const WorkerAttributeSet ACLAttributeSet = { SpatialConstants::ZoningAttribute };
```

Comment:
```suggestion
	const WorkerAttributeSet LoadBalancerAttributeSet = { SpatialConstants::ZoningAttribute };
```

Reply:
Also the `SpatialConstants::ZoningAttribute` is terribly named - implies it's a WorkerAttributeSet when it's just a string. Can you the constant to be a `WorkerAttributeSet ` (just wrap in braces and change type). I can only ever see it used like that in code anyway.

Reply:
Also after that point can probably just put `SpatialConstants::ZoningAttribute` in code instead of doing this assignment. Also, `LoadBalancing(er)Attribute` is probably a better name.

Reply:
Changed type, name to `LoadBalancerAttributeSet` and changed everywhere